src/chains/algorand_1/algoTransaction.ts

```diff
@@ -239,6 +243,7 @@ export class AlgorandTransaction implements Transaction {
       const chainTxHeaderParams = this._chainState.chainInfo.nativeInfo.transactionHeaderParams
       this._actionHelper.applyCurrentTxHeaderParamsWhereNeeded(chainTxHeaderParams, this.options)
       this._algoSdkTransaction = new AlgoTransactionClass(this._actionHelper.actionEncodedForSdk as any)
+      this._algoSdkTransaction.group = this._actionHelper.actionEncodedForSdk.group
```

Comment:
This line requires a clear comment. Group just seems out of place here so the comment should explain why its special and what's happening here